### PR TITLE
Microverse:portfolio-project:Animation added:Deployed

### PR DESCRIPTION
**Pull request by Shubh-Dev**
Dear Reviewer,
I just Deployed my Project. Everything is working fine on the web. let me know if any changes are required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Description-
 This Project is an initial experiment of creating an excellent personal portfolio for a Software developer. An excellent portfolio is one of the most effective tools for a software developer. It is the easiest way to showcase one's projects and abilities. The designs and the content of this project are tried to keep simple yet elegant.
 
-## Webpage Link - https://shubh-dev.github.io/portfolio-setup/
+## Webpage Link - https://shubh-dev.github.io/portfolio-setup
```
Comment (prolajumokeoni) [inline README.md]: - Kindly use this format for   your demo link: 
-
```suggestion
## Live demo 
 [Webpage link ](https://shubh-dev.github.io/portfolio-setup)
```

So it looks like this   [Webpage link ](https://shubh-dev.github.io/portfolio-setup)

**Comment by prolajumokeoni** [comment]
Hi @Shubh-Dev I noticed your logo in the mobile version spans to another line it would be nice to make it look like figma design 👍🏿 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
llogo  | Capture

**Review by prolajumokeoni — APPROVED** [review]
Hi @Shubh-Dev 👋🏿 
Your project is deployed!   Congratulations! 🎉 


all-done-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Shubh-Dev) [reply README.md]: @prolajumokeoni Hi, Thank you so much for approving my project and guiding me 👍 :-).

Comment (Shubh-Dev) [reply README.md]: @prolajumokeoni I need some more help. Can you please explain a bit about the Live Demo section you suggested? I don't have a Demo right now. Also can you please guide me, on how to create a demo?

Comment (prolajumokeoni) [reply README.md]: Live demo means live demonstration, this also means deployed link🤗

Comment (prolajumokeoni) [reply README.md]: I suggested you add that as a heading and the deployed link under, your heading can be anything

**Pull request merged**